src/component/PokemonCard.js

```diff
@@ -0,0 +1,44 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import styles from "../component/PokemonCard.css"
+
+const PokemonCard = ({url}) => {
+    const [pokemon, setPokemon] = useState([])
+
+    useEffect(() => {
+        async function getPokemonData() {
+
+            try {
+                const result = await axios.get( `${url}`);
+                console.log(result.data);
+                setPokemon(result.data);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+        getPokemonData()
+    }, [])
+
+    return (
+        <section className="pokemon-section">
+        <div id="pokemon-card">
+            {Object.keys(pokemon).length > 0 &&
```

Comment:
Hier kan het dan iets simpeler
```suggestion
            {pokemon &&
```